core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsGoogleProjectComponent.scala

```diff
@@ -0,0 +1,68 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.model.{RawlsBillingProjectName, RawlsGoogleProject}
+
+final case class RawlsGoogleProjectRecord(googleProjectId: String,
+                                          billingAccount: Option[String],
+                                          message: Option[String],
+                                          billingProjectId: String
+)
+
+object RawlsGoogleProjectRecord {
```

Comment:
There is such a thing as a `GoogleProject`, which is different than this specific table, so I've specified `RawlsGoogleProject` here, but not every file or method name has the "Rawls" qualifier - is this a good name?  Should absolutely everything to do with this table have "Rawls" in front of it?